scripts/format-grpc-sys

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+clang-format-5.0 -i grpc-sys/grpc_wrap.c
```

Comment:
can we remove -5.0 here?

Reply:
It's better to be kept, grpc explicitly states `-5.0` too. `clang-format`'s version changes all the time, ubuntu 18, it's 6. ubuntu 16, it's 3.8.